uk.co.mangobrain.Infector.appdata.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+    <id type="desktop">infector.desktop</id>
+	<metadata_license>CC0-1.0</metadata_license>
+	<project_license>GPL-3.0+</project_license>
+    <name>Infector</name>
+    <summary>Play the classic two- to four-player board game Ataxx</summary>
+    <description>
+        <p>
+            Infector is a version of the classic board games Ataxx and
+            Hexxagon. Play alone against simple AI, or with up to four
+            players, locally or over a network.
+        </p>
+    </description>
```

Comment:
Some screenshots would be nice

```
<screenshots>
		<screenshot type="default">
			<image type="source">http://flarerpg.org/wp-content/uploads/2018/03/empyrean_1.jpg</image>
                </screenshot>
</screenshots>
```

Reply:
Yeah, we need screenshots. Other than that, looks good to me.